maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -193,6 +187,8 @@ private function store() {
 
 		if ( !$srcStorage->exists( $remoteFile ) ) {
 			$this->output( "\tCannot find image to sync \n" );
+			$this->imageSyncQueue->setError( self::ERROR_CANT_FIND_FILE );
```

Comment:
`imageSyncQueue` is a single item. I do agree with you that this script is somehow misleading, but rewriting it is simply outside the scope of PLATFORM-841